src/js/components/Tip/__tests__/__snapshots__/Tip-test.tsx.snap

```diff
@@ -579,12 +579,6 @@ exports[`Tip themed 1`] = `
   animation-delay: 0.01s;
 }
 
-@media only screen and (max-width:768px) {
```

Comment:
I'm trying to understand why this margin goes away with these changes. Can you explain?